src/ui/public/modals/confirm_modal.html

```diff
@@ -1,32 +1,10 @@
-<div class="kuiModal" style="width: 450px" data-test-subj="confirmModal">
-  <div class="kuiModalHeader" ng-if="title">
-    <div class="kuiModalHeader__title" data-test-subj="confirmModalTitleText">
-      {{title}}
-    </div>
-  </div>
-  <div class="kuiModalBody">
-    <div
-      class="kuiModalBodyText"
-      data-test-subj="confirmModalBodyText"
-    >
-      {{message}}
-    </div>
-  </div>
-
-  <div class="kuiModalFooter">
-    <button
-      class="kuiButton kuiButton--hollow"
-      data-test-subj="confirmModalCancelButton"
-      ng-click="onCancel()"
-    >
-      {{cancelButtonText}}
-    </button>
-    <button
-      class="kuiButton kuiButton--primary"
-      data-test-subj="confirmModalConfirmButton"
-      ng-click="onConfirm()"
-    >
-      {{confirmButtonText}}
-    </button>
-  </div>
-</div>
+<confirm-modal
```

Comment:
Correct, and yea, it's a good point, but you have to create the angular directives that map to the react component, and I believe there can only be one.  So if the react confirm modals in a couple different places, which one is going to register the directive? @weltenwort does that sound accurate?

I tried to find the original discussion for where this model came from, and it appears in the very first `react icons` PR (https://github.com/elastic/kibana/pull/10247), but couldn't find the discussion.  Perhaps it was offline - @kjbekkelund or @cjcenizal, do you remember us discussing this?